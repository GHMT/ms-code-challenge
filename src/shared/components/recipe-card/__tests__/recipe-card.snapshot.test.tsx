import React from 'react';
import { shallow } from "enzyme";

import RecipeCard from '..';

describe('Recipe Card Component snapshots', () => {
  it('match snapshot', () => {
    const RecipeCardShallowed = shallow(
        <RecipeCard
            id='abc123'
            img=''
            title='Dummy Title'
            onRecipeClick={jest.fn(x => x)}
    />
    );

    expect(RecipeCardShallowed).toMatchSnapshot();
  });
});