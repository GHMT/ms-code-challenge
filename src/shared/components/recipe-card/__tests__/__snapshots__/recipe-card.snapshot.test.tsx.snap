// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipe Card Component snapshots match snapshot 1`] = `
<Fragment>
  <Card
    body={false}
    border="dark"
    className="recipeCard"
    onClick={[Function]}
  >
    <CardBody>
      <CardTitle>
        <Truncate
          ellipsis={
            <span>
              ...
            </span>
          }
          lines={3}
          trimWhitespace={false}
          width={0}
        >
          Dummy Title
        </Truncate>
      </CardTitle>
    </CardBody>
    <CardImg
      src=""
      variant="bottom"
    />
  </Card>
</Fragment>
`;
